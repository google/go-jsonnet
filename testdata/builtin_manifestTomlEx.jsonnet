local object = {
  foo: 'baz',
  abc: 'def',
  bar: self.foo,
  baz: 1,
  bazel: 1.42,
  boom: -1,
  bim: false,
  bam: true,
  blamo: {
    cereal: [
      '<>& fizbuzz',
      ['a', ['b']],
    ],

    treats: [
      {
        name: 'chocolate',
      },
    ],
  },
};

local object2 = {
  key: 'value',
  simple: { t: 5 },
  section: {
    a: 1,
    nested: { b: 2 },
    'e$caped': { q: 't' },
    array: [
      { c: 3 },
      { d: 4 },
    ],
    nestedArray: [{
      k: 'v',
      nested: { e: 5 },
    }],
  },
  arraySection: [
    { q: 1 },
    { w: 2 },
  ],
  'escaped"Section': { z: 'q' },
  emptySection: {},
  emptyArraySection: [{}],
  bool: true,
  notBool: false,
  number: 7,
  array: ['s', 1, [2, 3], { r: 6, a: ['0', 'z'] }],
  emptyArray: [],
  '"': 4,
};

{
  object: std.manifestTomlEx(object, '  '),
  object2: std.manifestTomlEx(object2, '  '),
}