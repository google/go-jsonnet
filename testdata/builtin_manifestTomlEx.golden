{
   "object": "abc = \"def\"\nbam = true\nbar = \"baz\"\nbaz = 1\nbazel = 1.4199999999999999\nbim = false\nboom = -1\nfoo = \"baz\"\n\n[blamo]\n  cereal = [\n    \"<>& fizbuzz\",\n    [ \"a\", [ \"b\" ] ]\n  ]\n\n  [[blamo.treats]]\n    name = \"chocolate\"",
   "object2": "\"\\\"\" = 4\narray = [\n  \"s\",\n  1,\n  [ 2, 3 ],\n  { a = [ \"0\", \"z\" ], r = 6 }\n]\nbool = true\nemptyArray = []\nkey = \"value\"\nnotBool = false\nnumber = 7\n\n[[arraySection]]\n  q = 1\n\n[[arraySection]]\n  w = 2\n\n[[emptyArraySection]]\n\n[emptySection]\n\n[\"escaped\\\"Section\"]\n  z = \"q\"\n\n[section]\n  a = 1\n\n  [[section.array]]\n    c = 3\n\n  [[section.array]]\n    d = 4\n\n  [section.\"e$caped\"]\n    q = \"t\"\n\n  [section.nested]\n    b = 2\n\n  [[section.nestedArray]]\n    k = \"v\"\n\n    [section.nestedArray.nested]\n      e = 5\n\n[simple]\n  t = 5"
}
